ogre_mesh_viewer.py

```diff
@@ -138,6 +138,7 @@ def __init__(self, app):
         self.show_metrics = False
         self.show_render_settings = False
         self.side_panel_visible = True
+        self.load_file_lock = False
```

Comment:
```suggestion
        self.is_filedialog_open = False
```
this is not a true lock, so dont give the impression